app/qml/form/editors/MMFormTextMultilineEditor.qml

```diff
@@ -38,7 +38,7 @@ MMBaseInput {
   property bool _fieldRememberValueState: parent.fieldRememberValueState
 
   property alias placeholderText: textArea.placeholderText
-  property alias text: textArea.text
+  property string text: _fieldValue === undefined || _fieldValueIsNull ? '' : _fieldValue
```

Comment:
Why not having this directly in the `textArea.text` property?

Reply:
if we use alias we get runtime warnings 
`qrc:/com.merginmaps/imports/MMInput/form/MMFormPage.qml:261:7: QML Loader: Binding loop detected for property "fieldValue"`

without alias it works 🤷